client/verta/verta/operations/monitoring/labels.py

```diff
@@ -12,23 +12,46 @@
 from verta._internal_utils import time_utils
 from .utils import maybe
 
-class Labels:
+
+class Labels(object):
+    """
+    Repository object for finding labels.
```

Comment:
I wonder if "Repository object" would be confusing, since our platform has an actual standalone concept of `Repository`. I'd favor "Collection object", but I'm not sure if there's a standard nomenclature for this sort of concept.

Reply:
[_Repository_](https://martinfowler.com/eaaCatalog/repository.html) might regrettably also be the most appropriate term for this object's role in the library. That said, we don't yet have a consistent framework or practice which we've settled upon in our library, so an alternative term can be found until we have a framework which forces the clarification of language.